app/org/sagebionetworks/bridge/services/SubpopulationService.java

```diff
@@ -41,11 +59,32 @@ public Subpopulation createSubpopulation(Study study, Subpopulation subpop) {
         Validator validator = new SubpopulationValidator(study.getDataGroups());
         Validate.entityThrowingException(validator, subpop);
         
+        // Create a default consent for this subpopulation.
+        StudyConsentView view = studyConsentService.addConsent(subpop, defaultConsentDocument);
+        studyConsentService.publishConsent(study, subpop, view.getCreatedOn());
+        
         return subpopDao.createSubpopulation(subpop);
     }
     
     /**
+     * Create a default subpopulation for a new study
+     * @param study
+     * @return
+     */
+    public Subpopulation createDefaultSubpopulation(Study study) {
+        SubpopulationGuid subpopGuid = SubpopulationGuid.create(study.getIdentifier());
+        // Migrating, studies will already have consents so don't create and publish a new one
+        // unless this is part of the creation of a new study after the introduction of subpopulations.
+        if (studyConsentService.getAllConsents(subpopGuid).isEmpty()) {
+            StudyConsentView view = studyConsentService.addConsent(subpopGuid, defaultConsentDocument);
+            studyConsentService.publishConsent(study, subpopGuid, view.getCreatedOn());
+        }
+        return subpopDao.createDefaultSubpopulation(study.getStudyIdentifier());
+    }
```

Comment:
Unit test for this? (For both cases, where consents already exist, and where consents don't already exist.)